core/modules/replica/tools/qserv-replica-mysql-test.cc

```diff
@@ -213,63 +283,158 @@ int main(int argc, const char* const argv[]) {
             "    CREATE_DATABASE <database>\n"
             "    DROP_DATABASE   <database>\n"
             "\n"
-            "    QUERY <file>\n"
+            "    QUERY      <file>\n"
+            "    QUERY_WAIT <file>\n"
             "\n"
             "Parameters:\n"
-            "  database             - the name of a database\n"
-            "  file                 - the name of a file from which to read a SQL statement.\n"
-            "                         If the file name is set to '-' then statement will be read\n"
-            "                         from the standard input stream.\n"
+            "\n"
+            "    <database> \n"
+            "\n"
+            "      the name of a database\n"
+            "\n"
+            "    <file> \n"
+            "\n"
+            "      the name of a file from which to read a SQL statement.\n"
+            "      If the file name is set to '-' then statement will be read\n"
+            "      from the standard input stream.\n"
+            "\n"
             "Flags and options:\n"
-            "  --no-auto-reconnect  - do *NOT* auto-reconnect to the service if its drops the connection\n"
-            "                         after a long period of the applicatin inactivity\n"
-            "  --no-transaction     - do *NOT* start/commit transactions when executing\n"
-            "                         database queries\n"
-            "  --no-result-set      - do *NOT* explore the result set after executing statements\n"
-            "  --host               - the DNS name or IP address of a host where the service runs\n"
-            "                         [ DEFAULT: localhost]\n"
-            "  --port               - the port number for the MySQL service\n"
-            "                         [ DEFAULT: 3306]\n"
-            "  --user               - the name of the MySQL user account\n"
-            "  --password           - a password to log into the MySQL user account\n"
-            "  --default-database   - the name of the default database to connect to\n"
-            "                         [ DEFAULT: '']\n");
+            "\n"
+            "    --db-allow-reconnect \n"
+            "\n"
+            "      change the default database connecton handling node. Set 0 to disable automatic\n"
+            "      reconnects. Any other number would man an opposite scenario.\n"
+            "      DEFAULT: " + std::to_string(::databaseAllowReconnect ? 1 : 0) + "\n"
+            "\n"
+            "    --db-reconnect-timeout \n"
+            "\n"
+            "      change the default value limiting a duration of time for making automatic\n"
+            "      reconnects to a database server before failing and reporting error (if the server\n"
+            "      is not up, or if it's not reachable for some reason)\n"
+            "      DEFAULT: " + std::to_string(::databaseConnectTimeoutSec) + "\n"
+            "\n"
+            "    --db-max-reconnects\n"
```

Comment:
Again, it is called `--db-max-retries` above.

Reply:
Fixed